lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -1367,11 +1152,9 @@ public function getDropDatabaseSQL($database)
      *
      * @param Table|string $table
```

Comment:
Not directly related, but we should take care of these unions in a major version.

Reply:
I'd like to but AFAIK it may complicate some cases. Should be resolved on a per-case basis.